doc/src/manual/arrays.md

```diff
@@ -764,9 +786,12 @@ julia> speye(3,5)
   [3, 3]  =  1.0
 ```
 
-The [`sparse()`](@ref) function is often a handy way to construct sparse matrices. It takes as
-its input a vector `I` of row indices, a vector `J` of column indices, and a vector `V` of stored
-values. `sparse(I,J,V)` constructs a sparse matrix such that `S[I[k], J[k]] = V[k]`.
+The [`sparse()`](@ref) function is often a handy way to construct sparse arrays. For
+matrices it takes as its input a vector `I` of row indices, a vector `J` of column
```

Comment:
Perhaps generalize the description of `sparse`? For example, `sparse` also accepts dimension and combination arguments, and altogether alternatively accepts a dense array to transform to a sparse equivalent